.simplecov

```diff
@@ -0,0 +1,10 @@
+require 'simplecov-console'
```

Comment:
[simplecov-console](https://github.com/chetan/simplecov-console) will generate a test coverage report in your terminal when running the test suite with `bundle exec rake test`.

<img width="1102" alt="screen shot 2018-01-25 at 1 28 17 pm" src="https://user-images.githubusercontent.com/73866/35405270-9e494640-01d3-11e8-896c-895add60e123.png">